tex/gregoriotex-syllable.tex

```diff
@@ -818,7 +822,7 @@
 }%
 
 \newif\ifgre@newbarspacing%
-\gre@newbarspacingfalse%
+\gre@newbarspacingtrue%
```

Comment:
For future reference:

The fact that you changed the default and implemented the fix at the same time threw me for a bit, especially since all the test changes were accepted simultaneously in gregorio-test.  I had deliberately been holding off on changing the default on my feature/bar_spacing branch in order to keep it clear (to myself at least) what changes resulted from what fixes.  I, for one, would very much appreciate separate commits for this sort of thing, even in cases where you know that the expectations being accepted as a result of the change in default behavior are wrong because they reveal a bug.